src/ui/pages/projects/components/ProjectList/ListItem.stories.tsx

```diff
@@ -49,7 +49,7 @@ export const FailSync = {
   args: {
     onSync: flow(
       syncProject,
-      taskEither.mapLeft(() => syncExceptionADT.projectDirMissing()),
+      taskEither.mapLeft(() => syncExceptionADT.invalidFilename('SB_INVALID_FILENAME')),
```

Comment:
What does the “SB” part mean?

Reply:
StoryBook, so we don't have to guess where this file name is coming from.

Reply:
Aha! Of course :)